components/dashboard/src/projects/NewProject.tsx

```diff
@@ -140,11 +140,13 @@ export default function NewProject() {
             cloneUrl: repo.cloneUrl,
             account: repo.account,
             provider,
+            ...(User.is(teamOrUser) ? { userId: teamOrUser.id } : { teamId: teamOrUser.id }),
             appInstallationId: String(repo.installationId),
-            teamId: team.id
         });
 
-        history.push(`/${team.slug}/${repo.name}/configure`);
+        history.push(User.is(teamOrUser)
+            ? '/projects' // FIXME
```

Comment:
*Q*: shouldn't it be `/projects/repoName/configure`?

Reply:
A: Yes, good catch!

Reply:
@AlexTugarev So, like this?

- Individuals:
  - `/projects`
  - `/projects/:projectName/configure`
- Teams
  - `/:teamSlug/projects`
  - `/:teamSlug/:projectName/configure`

(I think this is slightly inconsistent in some sense, but doesn't shock me 🤷‍♂️)